examples/simulation/Tutorial1/macros/run_tutorial1.C

```diff
@@ -160,4 +158,34 @@ void run_tutorial1(Int_t nEvents = 10,
     cout << "Macro finished successfully." << endl;
 
     // ------------------------------------------------------------------------
+
+    TChain chain("cbmsim");
+    if (run->IsMT()) {
+        cout << endl << "MT mode, reading two files: t1 and t2" << endl;
+        chain.Add((outFile.replace(outFile.find(".root"), 5, "_t1.root")).c_str());
+        chain.Add((outFile.replace(outFile.find("_t1.root"), 8, "_t2.root")).c_str());
```

Comment:
Is Geant4 always creating two threads? Just wondering.

Reply:
Currently yes, no idea if there are plans for more...

Reply:
@karabowi,

does it make sense to have the check in the same macro? Wouldn't it be better to have the check in another macro?

Reply:
@fuhlig1 the checking of the output file belongs to this macro, in my opinion.
Why should we create another macro just to check the output file?

Reply:
In my opinion you check here two things at once. First you check if you are able to run the multithreaded Geant4 and second you check if you can read the results of the simulation. For me this are two distinct tests but I am also fine to keep the macro as it is.